lib/blockscore/base.rb

```diff
@@ -53,21 +54,25 @@ def self.resource
       @resource ||= Util.to_underscore(to_s.split('::').last)
     end
 
-    def self.api_url
-      'https://api.blockscore.com/'
-    end
-
     def self.endpoint
       fail NotImplementedError, ABSTRACT_WARNING if equal?(Base)
 
-      "#{api_url}#{Util.to_plural(resource)}"
+      Pathname(Util.to_plural(resource))
+    end
+
+    def member_endpoint
+      endpoint + id
```

Comment:
Yes - the tests all pass, and in my RSpec refactoring there will be specific tests in Base. The endpoint is a Pathname so adding segments joins with the slash.